src/sentry/models/event.py

```diff
@@ -384,41 +383,6 @@ def get_level_display(self):
         else:
             return None
 
-    # deprecated accessors
-
-    @property
-    def logger(self):
```

Comment:
Could these be used in plugins?

Reply:
I checked the sentry-plugins repo (well there are only 3 in there now most of the plugins were migrated here) and they don't have any usage of these